hmpps-prisoner-search-indexer/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/listeners/DomainEventListenerIntTest.kt

```diff
@@ -21,7 +23,8 @@ class DomainEventListenerIntTest : IntegrationTestBase() {
     prisonApi.stubOffenders(PrisonerBuilder(prisonerNumber = prisonerNumber))
 
     hmppsDomainSqsClient.sendMessage(
-      SendMessageRequest.builder().queueUrl(hmppsDomainQueueUrl).messageBody(validIepMessage(prisonerNumber)).build(),
+      SendMessageRequest.builder().queueUrl(hmppsDomainQueueUrl)
+        .messageBody(validRPMessage(prisonerNumber, "restricted-patients.patient.added")).build(),
```

Comment:
had to switch this test from using incentive message which now works differently, to restricted-patients